src/00/flash.asm

```diff
@@ -195,6 +196,7 @@ _:  pop af
     jr nz, .loop
     jp .return
 .ram_end:
+.echo "writeFlashBuffer RAM footprint: 0x{0:X4}" .ram_end-.ram
```

Comment:
My bad. I know what I did, I'll fix it. I'd typed out what caused it and accidentally hit cancel instead of comment and don't feel like putting it all here again, though I will if you want me to.